test/templates/alert/data.json

```diff
@@ -0,0 +1,9 @@
+{
+    "properties": {
+        "text": "Alert Text",
+        "action": "#/action"
```

Comment:
You're not explicitly passing an action here so probably a boolean like "dismissible" would work — some flag to either include the close or not. See https://getbootstrap.com/docs/4.0/components/alerts/#dismissing. 

But I believe currently ALL of our alerts can be dismissed so for MVP it's OK.